src/fu-tool.c

```diff
@@ -1614,6 +1615,17 @@ fu_util_hwids (FuUtilPrivate *priv, gchar **values, GError **error)
 		FU_HWIDS_KEY_BASEBOARD_PRODUCT,
 		NULL };
 
+	/* secret way to override SMBIOS for debugging */
```

Comment:
Why not an argument with an XML or JSON payload?  It seems like it's cleaner...

Reply:
Similarly, maybe it's worth being able to dump "current" values as fwupd sees, so you can just change one?
I guess I don't know the underlying use for this - so if that's a lot of work without a lot of utility speak up.

Reply:
> Similarly, maybe it's worth being able to dump "current" values as fwupd sees

Makes sense. I added `import-hwids` and `export-hwids` although I'm not completely sure about the verbs in both. Better ideas welcome.

Reply:
I think those are fine, but you probably want the old `hwids` command to rename to match too.  `print-hwids`?

Reply:
Nah, I merged the import hwids action into the existing one; it's easy enough to detect.